### ISSUE TITLE
Tab panel for Between the Lines grid block

### ISSUE THREAD

**Issue by herohousecreative** [rkrussel/nbba-website#3]
The tab panel will give users the option to toggle between three different media types: Photos, Videos and Podcasts.

Must use ARIA, including aria-selected and roles such as "tab" and "panel" to support robust accessibility needs.

There is a current solution implemented, but i suspect the script is bad. I suck at JS!

Pertinent files include: front-page.php (for HTML), /lib/custom.css (for CSS) and /js/a11-tablist.js (for JS.)

**Comment by herohousecreative** [comment]
FYI: This is where I got the code to construct this panel: http://oaa-accessibility.org/example/34/

**Comment by herohousecreative** [comment]
Just an update... Still can't get this to work. I'm a dunce with JS - and that's a huge component here.

I tried this example (literally copying the demo code directly into the theme), and it still didn't work. Example is at: https://www.deque.com/blog/a11y-support-series-part-1-aria-tab-panel-accessibility/

Thoughts?

**Comment by rkrussel** [comment]
So when I click the tabs, it works, the header changes to whichever tab you select.  Do we just need to.  Is the issue that we need to assign the Aria attributes to the generated elements?

**Comment by herohousecreative** [comment]
Actually… thought I’d closed this issue. I resolved this a little earlier and will push those changes ASAP.

I’ve done quite a bit of work tonight – including a lot of CSS. Will commit and get these online in the next few.

I assume the database is working on your end now?

Thanks dude,
Justin

**Comment by rkrussel** [comment]
Looks like the database is working, I think I did everything right the other day but I think my browser had the broken version cached. So I just logged on today and it worked!

So now that I've finally had time,  I've mostly been playing around trying to make sense of everything going on.  I thought wordpress was supposed to be easy but this is making me feel like a novice.  I was doing a little bit of styling, but if you're working on that I'll let you push yours first so I can blow away anything of mine that might conflict.
